components/TodoItemButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { Pencil, Trash } from "@tamagui/lucide-icons";
+import { useContext } from "react";
+import { Button } from "tamagui";
+import { observer } from "mobx-react-lite";
+
+import { Todo, TodoContext } from "../stores/Todo.store";
+
+type GenericProps = {
+  icon: JSX.Element;
+  onPress: () => void;
+  accessibilityLabel: string;
+};
+
+function ItemButton({ icon, onPress, accessibilityLabel }: GenericProps) {
+  return (
+    <Button
+      accessibilityLabel={accessibilityLabel}
+      backgroundColor="$white1"
```

Comment:
in general are you using/following 'dark'/'light' theme in your app? Or does it have static theme?
If you use dark/light theme, then the colours should be picked up from the 'theme' and not static values.

Just a note.

Reply:
I'm using a static theme